crawler.js

```diff
@@ -679,6 +668,7 @@ export class Crawler {
       puppeteerOptions: this.puppeteerArgs,
       crawlState: this.crawlState,
       screencaster: this.screencaster,
+      healthChecker: this.healthChecker,
```

Comment:
Should remove `timeout` here as well, yeah?

Reply:
(it wouldn't let me select the actual `timeout` line for a comment as it's not in the diff for this PR)